.github/workflows/SDK-Suppressions-Label.yaml

```diff
@@ -0,0 +1,114 @@
+name: SDK Suppression Label
+
+on:
+  pull_request:
+    branches:
+      - main
+      - RPSaaSMaster
+      - release*
+    types:
+      - opened
+      - synchronize
+      - reopened
```

Comment:
```suggestion
```

Simplify by removing this, since these are already the default values:

https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request

Reply:
removed